File: Python/PRAK504-2210817310017-Rohid Akmal.py
def reverse (a):
    reverse = 0 
    while a != 0:
        b = a % 10
        reverse = reverse * 10 + b
        a //= 10
    return reverse

for i in range(3):
    A, B = map(int,input().split())
    A = reverse(A)
    B = reverse(B)
    c = A + B
    print("%d\n"%(reverse(c)))

File: C/PRAK504-2210817310017-Rohid_Akmal.c
#include <stdio.h>

int reverse(int a){
    int b, reverse = 0 ;
    while (a != 0){
        b = a % 10;
        reverse = reverse * 10 + b;
        a /= 10;
    }
    return reverse;
}

int main(){ 
    int A, B;
    scanf("%d %d",&A,&B); 
    A=reverse(A); 
    B=reverse(B);
    int C = A+B; 
    printf("%d",reverse(C));
}